packages/web-app-files/src/helpers/resources.ts

```diff
@@ -442,7 +442,7 @@ function _buildLink(link): Share {
     permissions: link.permissions,
     description,
     stime: link.stime,
-    name: typeof link.name === 'string' ? link.name : '',
+    name: typeof link.name === 'string' ? link.name : (link.token as string),
```

Comment:
what's the reason to use link.token as a fallback instead of the empty string like before?